packages/common/src/api/tan-query/queryKeys.ts

```diff
@@ -1,11 +1,13 @@
 export const QUERY_KEYS = {
+  accountUser: 'accountUser',
```

Comment:
todo: consider optimistic updates on self user edit

ok to create follow up ticket for this one

Reply:
yeah i think the fear is that any incoming user data that happens to be the current user may clobber account specific data like playlist-library